src/compiler/crystal/tools/doc/item.cr

```diff
@@ -6,4 +6,68 @@ module Crystal::Doc::Item
   def formatted_summary
     @generator.summary(self)
   end
+
+  macro def_to_json(type, properties)
+    Item.def_to_json(nil, {{properties}})
```

Comment:
No I meant this specific override, it just seems that the third `macro def_to_json(type, properties)` below will override this definition entirely.